quarkchain/cluster/cluster_config.py

```diff
@@ -455,7 +458,22 @@ def from_dict(cls, d):
         config.QUARKCHAIN = QuarkChainConfig.from_dict(config.QUARKCHAIN)
         config.MONITORING = MonitoringConfig.from_dict(config.MONITORING)
         config.MASTER = MasterConfig.from_dict(config.MASTER)
-        config.SLAVE_LIST = [SlaveConfig.from_dict(s) for s in config.SLAVE_LIST]
+        temp_GRPC_SLAVE_LIST = []
```

Comment:
at this step, should already have `config.SLAVE_LIST` as a list of dicts with type `QKCRPC` and `config.GRPC_SLAVE_LIST` as a list of dicts with type `GRPC`, right? do we need to filter again?

Reply:
this step is to solve future type transformation loading from command line as we discussed before. if I remove it, GRPC_LIST will be load as dict rather than object. However, if we just think about loading from config file, then we don't need this step.

Reply:
why not just 

```
config.SLAVE_LIST = [SlaveConfig.from_dict(s) for s in config.SLAVE_LIST]
config.GRPC_SLAVE_LIST = [SlaveConfig.from_dict(s) for s in config.GRPC_SLAVE_LIST]
```